conf/test_experimental.config

```diff
@@ -28,7 +28,7 @@ params {
     matrix =  'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/rnaseq_expression/SRP254919.salmon.merged.gene_counts.top1000cov.tsv'
     contrasts =  'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/rnaseq_expression/SRP254919.contrasts.csv'
     tools = "$projectDir/assets/tools_samplesheet.csv"
-    pathway = "diff_prop,diff_prop_noperm,filtered_pcor,prop"
+    pathway = "propd"
```

Comment:
I think the test should be as exhaustive as possible, i.e. test as many pathways as possible